modules/ossm-installing-the-istioctl-tool.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-the-istioctl-tool_{context}"]
+= Installing the Istioctl tool
+
+.Prerequisites
+
+* You have access to the {ocp-product-title} web console.
+
+.Procedure
+
+. In the {ocp-product-title} web console, click the question mark icon and select *Command Line Tools*.
+
+. Click *Download istioctl*. Choose the version and architecture that matches your system.
+
+* link:https://mirror.openshift.com/pub/cgw/servicemesh/1.23.0/[Version 1.23.0]
+
+* link:https://mirror.openshift.com/pub/cgw/servicemesh/1.24.1/[Version 1.24.1]
```

Comment:
There will be another published version (1.23.3) while we will release GA.  
Would it make sense to also add the `latest` version link?

Reply:
@bmangoen and @pbajjuri20 the PR has been updated to include a link to latest.

When you have a moment, please review. After LGTM comments are added, I can move the PR to the next phase of the release process. Thanks!